lib/mp4/transmuxer.js

```diff
@@ -1248,6 +1249,11 @@ Transmuxer = function(options) {
     // Start at the top of the pipeline and flush all pending work
     this.transmuxPipeline_.headOfPipeline.flush();
   };
+
+  this.seeked = function() {
+    this.transmuxPipeline_.captionStream.resetDtsTracking();
```

Comment:
This should probably be wrapped in an `if (this.transmuxPipeline_.captionStream)` since `captionStream` won't exist if an AAC pipeline is setup.